experimental/dds/tree2/src/feature-libraries/editable-tree-2/proxies/node.ts

```diff
@@ -69,24 +69,24 @@ export interface NodeApi {
  */
 export const nodeApi: NodeApi = {
 	schema: (node: SharedTreeNode) => {
-		return assertTreeNode(node).schema;
+		return getTreeNode(node).schema;
```

Comment:
Optional: I've been thinking about renaming 'getTreeNode' and similar to 'getEditNode' since we're increasingly using "tree node" to refer to the proxy.

(Just mentioning in case you agree and wanted to bundle that into your change.  No worries if not.)

Reply:
Sure, I'll do the renames in here, why not :)